### PR TITLE
READY: Fix total traffic bug

### LINKED ISSUE #347: [3] Node balance is different in focus from when not in focus
Hover a node to show its balance. Then click it to focus on it. Now hover again and you may see (in some cases) a different balance.

### PR DESCRIPTION
**Pull request by MaxVanDeursen**
In this commit, the bug with the total traffic changing when focusing
the node has been addressed and fixed.

Fixes #347

### CODE REVIEW

File: Tribler/community/triblerchain/community.py [hunk 1]
```diff
@@ -189,8 +189,14 @@ def get_node(self, public_key, nodes, total_up=None, total_down=None):
             if total_up and total_down:
                 return {"public_key": public_key, "total_up": total_up, "total_down": total_down}
             else:
-                total_traffic = self.persistence.total_traffic(public_key)
-                return {"public_key": public_key, "total_up": total_traffic[0], "total_down": total_traffic[1]}
+                binary_public_key = public_key
+                total_traffic = self.persistence.total_traffic(binary_public_key)
```
Comment (ghabbenjansen) [inline Tribler/community/triblerchain/community.py]: `total_traffic` in the aggregates database takes a hexlified public key

File: Tribler/community/triblerchain/community.py [hunk 2]
```diff
@@ -189,8 +189,14 @@ def get_node(self, public_key, nodes, total_up=None, total_down=None):
...
+                latest_block = self.persistence.get_latest(binary_public_key)
+                if latest_block:
```
Comment (ghabbenjansen) [inline Tribler/community/triblerchain/community.py]: Are we using the latest block in the trustchain or are we using the sum of all edges in the aggregates database?

Comment (MaxVanDeursen) [inline Tribler/community/triblerchain/community.py]: That is what I was wondering as well. We'll discuss this in the team.

Comment (MaxVanDeursen) [inline Tribler/community/triblerchain/community.py]: We will not be using the latest block for this calculation.

File: Tribler/community/triblerchain/community.py [hunk 3]
```diff
@@ -171,12 +171,12 @@ def get_node(self, public_key, nodes, total_up=None, total_down=None):
         Get a node in an encoded format and with the maximum values given the current dictionary of nodes.
 
         The format is described as follows:
-            {"public_key": public_key, "total_up": total_up, "total_down": total_down}
-        This function checks whether the given total up and downloaded amounts are higher than the current recorded (if
-        any). Moreover, if the public key does not exist in the nodes list and no total_up and total_down is given, new
-        data is retrieved from the database.
+            { "public_key": public_key, "total_up": total_up, "total_down": total_down }
```
Comment (TimSpeelman) [inline Tribler/community/triblerchain/community.py]: Is this the correct formatting? Space after {

**Pull request merged**